odl/contrib/tomo/elekta.py

```diff
@@ -0,0 +1,198 @@
+# Copyright 2014-2017 The ODL contributors
+#
+# This file is part of ODL.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public License,
+# v. 2.0. If a copy of the MPL was not distributed with this file, You can
+# obtain one at https://mozilla.org/MPL/2.0/.
+
+"""Tomography helpers for Elekta systems."""
+
+import numpy as np
+import odl
+
+
+__all__ = ('elekta_icon_geometry', 'elekta_icon_space',
+           'elekta_icon_fbp')
+
+
+def elekta_icon_geometry(sad=780.0, sdd=1000.0,
+                         piercing_point=(390.0, 0.0),
+                         angles=np.linspace(1.2, 5.0, 332),
+                         detector_shape=(780, 720)):
+    """Tomographic geometry of the Elekta Icon CBCT system.
+
+    See the [whitepaper]_ for specific descriptions of each parameter.
+
+    All measurments are given in millimeters unless otherwise stated.
+
+    Parameters
+    ----------
+    sad : float, optional
+        Source to Axis distance.
+    sdd : float, optional
+        Source to Detector distance.
+    piercing_point : sequence of foat, optional
+        Position in the detector (in pixel coordinates) that a beam from the
+        source, passing through the axis of rotation at a straight angle,
+        hits.
+    angles : array-like, optional
+        List of angles that the projection images were taken at.
+        Given in radians.
+    detector_shape : sequence of int, optional
+        Shape of the detector (in pixels). Useful if a sub-sampled system
+        should be studied.
+
+    Returns
+    -------
+    elekta_icon_geometry : `CircularConeFlatGeometry`
+
+    Examples
+    --------
+    Create default geometry:
+
+    >>> from odl.contrib import tomo
+    >>> geometry = tomo.elekta_icon_geometry()
+
+    Use a smaller detector (improves efficiency):
+
+    >>> small_geometry = tomo.elekta_icon_geometry(detector_shape=[100, 100])
+
+    See Also
+    --------
+    elekta_icon_space : Default reconstruction space for the Elekta Icon CBCT.
+    elekta_icon_fbp: Default reconstruction method for the Elekta Icon CBCT.
+
+    References
+    ----------
+    .. [whitepaper] *Design and performance characteristics of a Cone Beam
+       CT system for Leksell Gamma Knife Icon*
```

Comment:
Some more info here? Where published or where available? One way to do this would be to make the text `[whitepaper]` (formatting also off here) a hyperlink and have two items for it in the `References` section, one for the link and one like now, just without the `.. ` so that the section is actually printed. Like so:
```
See the `[whitepaper]`_ for specific descriptions of each parameter.

...

References
----------
.. _[whitepaper]: <insert URL here>
[whitepaper] *Design and performance characteristics of a Cone Beam
    CT system for Leksell Gamma Knife Icon*
```